File: Web.UI/ClientApp/models/users/userListData.ts
import { User } from "./user";
import { TypeHelper } from "../../helpers/typeHelper";

export class UserListData {

    /// <summary>
    /// Top 100 items only
    /// </summary>
    public readonly List: User[] = [];

    /// <summary>
    /// If there are more than 100 returned items
    /// </summary>
    public readonly TooMuchData: boolean = false;
}

File: Service.Models/Users/UserListData.cs
using static Microsoft.EntityFrameworkCore.DbLoggerCategory;

namespace Toptal.BikeRentals.Service.Models.Users
{
    public sealed class UserListData : Model
    {
        /// <summary>
        /// Top 100 items only
        /// </summary>
        public Service.Models.Users.User[] List { get; set; }

        /// <summary>
        /// If there are more than 100 returned items
        /// </summary>
        public bool TooMuchData { get; set; }
    }
}